ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/observer/UtxoLedgerTokenStateObserver.java

```diff
@@ -82,5 +83,5 @@
      * @return Returns a {@link UtxoToken}.
      */
     @NotNull
-    UtxoToken onCommit(@NotNull T state);
+    UtxoToken onCommit(@NotNull T state, @NotNull DigestService digestService);
```

Comment:
Similar to above, does this need to be an argument here or should the service just be injected into the implementation?

Reply:
I think this can't be injected to implementation because CorDapp developers are to implement this one. @owenstanford could you please confirm?

Reply:
That's correct, this is user code executed in the DB sandbox which doesn't support service injection like the flow sandbox, we could add it of course, but this is probably a reasonable option.